tests/Unit/Numerical/RootFinding/Test_QuadraticEquation.cpp

```diff
@@ -0,0 +1,15 @@
+
+#include <catch.hpp>
+
+#include "Numerical/RootFinding/QuadraticEquation.hpp"
+
+TEST_CASE("Unit.Numerical.RootFinding.PositiveRoot",
+          "[Numerical][RootFinding][Unit]") {
+  CHECK(Approx(6.31662479035539985) == positive_root(1.0, -6.0, -2.0));
+}
+
+TEST_CASE("Unit.Functors.RealRoots", "[Numerical][RootFinding][Unit]") {
+  auto roots = real_roots(2.0, -11.0, 5.0);
+  CHECK(Approx(0.5) == roots[0]);
+  CHECK(Approx(5.0) == roots[1]);
+}
```

Comment:
add a test that has two equal roots
add a test that has one positive, one real root
add a test that has no real roots and triggers the assert